spec/models/inventory_spec.rb

```diff
@@ -1,107 +1,116 @@
 require 'rails_helper'
 include InventoryExceptions
-
-RSpec.describe Inventory, type: :model do
-  before :all do # this is only for the mocks!
-    @uuid = 'not real uuid'
+# these tests must be run in order and all at once
```

Comment:
Because the test creates the item type, item and then a reservation in
order.
On Apr 19, 2016 1:36 PM, "David Faulkenberry" <notifications@github.com>
wrote:

> In spec/models/inventory_spec.rb
> <https://github.com/umts/probable-engine/pull/48#discussion_r60275361>:
>
> > @@ -1,107 +1,116 @@
> >  require 'rails_helper'
> >  include InventoryExceptions
> > -
> > -RSpec.describe Inventory, type: :model do
> > -  before :all do # this is only for the mocks!
> > -    @uuid = 'not real uuid'
> > +# these tests must be run in order and all at once
>
> Out of curiosity, why is this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/umts/probable-engine/pull/48/files/ff532da7b9d3632ab2ab5cff73e8dcf0c4bc0d20#r60275361>
>